uint8 HOUGH=0
uint8 orientation_type
---
bool oriented
---
bool orienting